modules/mod_ginger_rdf/support/sparql_client.erl

```diff
@@ -68,4 +95,4 @@ decode(Data) ->
     Data.
 
 headers() ->
-    #{<<"Accept">> => <<"application/ld+json">>}.
+    #{<<"Accept">> => <<"application/json">>}.
```

Comment:
Should we make this: `<<"application/ld+json,application/json;q=0.9">>`?

Reply:
Yes, better. I’ll have to test that header against the DBpedia endpoint first.

Reply:
Unfortunately Virtuoso doesn’t get Accept headers properly, so had to work around this.